lib/core/remote_config/page/soft_update_page.dart

```diff
@@ -50,7 +46,7 @@ class _SoftUpdatePageState extends State<SoftUpdatePage> {
               Padding(
                 padding: const EdgeInsets.symmetric(horizontal: 87.0),
                 child: Text(
-                  widget.message,
+                  message,
```

Comment:
context.localizations.update_app_text

Reply:
decide message read from constructor or from localizations directly

Reply:
final String? message;
I make it optional and I put the default value from localizations directly
                {  message ?? context.localizations.update_app_text, }